.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,10 @@
+const prettier = require('prettier');
```

Comment:
As indicated in [eslint-doc-generator's doc](https://github.com/bmish/eslint-doc-generator#prettier)